content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of the webhooks, to
+manage their configurations.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed. In the installation,
```

Comment:
What installation guide do readers need to follow? What configuration profile should they use? Please assume folks have deployed Istio using one of our existing configuration profiles and provide them here with the steps they need to take to modify said deployment before starting the task.

Reply:
Revised as follows (the installation guide reuses the one in the /docs/tasks/security/dns-cert):

Install Istio with [DNS certificates configured](/docs/tasks/security/dns-cert) and
[`global.operatorManageWebhooks`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) set to `true`.